.github/workflows/on_merge_to_main.yml

```diff
@@ -34,9 +34,9 @@ jobs:
         id: yarn_cache_id
         with:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-yarn-4-${{ hashFiles('**/yarn.lock') }}
```

Comment:
I'm just busting the yarn cache in github actions to make sure it is full tested